src/hubspot-provider.ts

```diff
@@ -0,0 +1,101 @@
+const Pkg = require('../package.json');
+
+const Hubspot = require('@hubspot/api-client')
+
+
+type HubspotProviderOptions = {}
+
+function HubspotProvider(this: any, options : HubspotProviderOptions) {
+  const seneca:any = this
+
+  const entityBuilder = this.export('provider/entityBuilder')
+
+  seneca.message('sys:provider,provider:hubspot,get:info', get_info)
+  async function get_info(this: any, _msg: any) {
```

Comment:
add a line to separate here
if more messages are added later, we want all the messages to be together
see: https://github.com/mgp/book-notes/blob/master/the-non-designers-design-book.markdown - proximity